documentation/src/main/asciidoc/stories/assembly_virtual_threads.adoc

```diff
@@ -0,0 +1,20 @@
+[id='virtual-threads-{context}']
+= Virtual Threads Support
+
+This release introduces support for virtual threads, which can significantly improve application responsiveness and scalability under high concurrency.
```

Comment:
Do we want to say "This release"? Maybe just "Infinispan supports virtual threads..."?